java-grpc/flatbuffers-java/src/main/java/com/google/flatbuffers/Constants.java

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
Sorry, we cannot have this kind of duplication, they'll have to be pulled from their original location.

Reply:
My intent was to consolidate all Java client codec in a single directory and build and test with a single maven project. In that case you would delete the existing Java folder and there would be no duplication.
If you don’t like it, where do you suggest to put pom.xml of flatbuufers-java-grpc? Also, once you have 2 or more jars where one depends on another you really need a parent pom file to keep things consistent. That’s what I achieved with my multi-module maven project.
As for your comment about publishing flatc for multiple platforms - that is the hardest problem and I expected the difficulties. But we really need to figure this out. Otherwise we are back in the woods with developer needing to get flatc on his own

Reply:
It be preferable to keep things in existing locations, and keep things testable without Maven. I do not know what the requirements are for pom files.. most build systems I know can address files from anywhere, is that not the case in Maven?